site/dat/docs/ResultsLoader.md

```diff
@@ -0,0 +1,26 @@
+# Results Loader Executor
+
+Taurus provides `external-results-loader` executor for cases, where you want to load test
+results from existing report files generated outside of Taurus control (JTL files from JMeter, `simulation.log` files
+from Gatling, etc).
+
+For example, it can be used to quickly create BlazeMeter report for test data after the test.
```

Comment:
This sentence needs polish at the end of it. Like 'create BM report from test result files that you obtained from your colleague.